.github/workflows/tests-nat46x64.yaml

```diff
@@ -0,0 +1,233 @@
+name: Cilium L4LB NAT-46x64 (ci-nat46x64)
```

Comment:
Have you looked how to avoid the boilerplate with test-l4lb.yaml? Maybe we could add to the test-l4lb an additional step to run the tests?

Reply:
Given the completely different setup, yaml, agent and routing configs, I went for new one which felt cleaner.